src/modules/Builder/features/ActivityItems/ItemConfiguration/InputTypeItems/PhrasalTemplate/PhrasalTemplate.utils.ts

```diff
@@ -0,0 +1,77 @@
+import { v4 } from 'uuid';
+
+import i18n from 'i18n';
+import { Item, PhrasalTemplateField, PhrasalTemplateFieldType } from 'redux/modules';
+import { ItemResponseType } from 'shared/consts';
+
+// TODO: M2-7211 — Additional response types to be added with
+// additional UI for selecting discrete response values.
+const PHRASAL_TEMPLATE_COMPATIBLE_RESPONSE_TYPES = [
+  ItemResponseType.Date,
+  ItemResponseType.MultipleSelection,
+  ItemResponseType.NumberSelection,
+  ItemResponseType.SingleSelection,
+  ItemResponseType.Slider,
+  ItemResponseType.Text,
+  ItemResponseType.Time,
+  ItemResponseType.TimeRange,
+  // ItemResponseType.ABTrails,
+  // ItemResponseType.Flanker,
+  // ItemResponseType.MultipleSelectionPerRow,
+  // ItemResponseType.SingleSelectionPerRow,
+  // ItemResponseType.SliderRows,
+  // ItemResponseType.StabilityTracker,
+  // ItemResponseType.TouchPractice,
+  // ItemResponseType.TouchTest,
+];
+
+const DEFAULT_PHRASE = {
+  image: '',
```

Comment:
`null` is a more appropriate default value for `image`, since it represents "no image" rather than "an image with the URL of an empty string". By strongly typing this constant, you also get the TypeScript intellisense that `null` is an expected possible value for `image`:
```suggestion
const DEFAULT_PHRASE: PhrasalTemplateResponseValues['phrases'][number] = {
  image: null,
```